src/components/tooltip/TrackPower.js

```diff
@@ -167,25 +195,7 @@ class TooltipTrackPowerImpl extends React.PureComponent<Props> {
             'TrackPower--tooltip-power-milliwatt'
           )}
           {previewSelection.hasSelection
-            ? this._formatPowerValue(
-                this._computePowerSumForPreviewRange(previewSelection),
-                'TrackPower--tooltip-energy-carbon-used-in-preview-kilowatthour',
-                'TrackPower--tooltip-energy-carbon-used-in-preview-watthour',
-                'TrackPower--tooltip-energy-carbon-used-in-preview-milliwatthour',
-                'TrackPower--tooltip-energy-carbon-used-in-preview-microwatthour'
-              )
-            : null}
-          {previewSelection.hasSelection
-            ? this._formatPowerValue(
-                (1000 /* ms -> s */ *
-                  3600 /* s -> h */ *
-                  this._computePowerSumForPreviewRange(previewSelection)) /
-                  (previewSelection.selectionEnd -
-                    previewSelection.selectionStart),
-                'TrackPower--tooltip-average-power-kilowatt',
-                'TrackPower--tooltip-average-power-watt',
-                'TrackPower--tooltip-average-power-milliwatt'
-              )
+            ? this.renderForPreviewSelection(previewSelection)
```

Comment:
You can remove the test for previewSelection.hasSelection here, it is already done at the beginning of renderForPreviewSelection.